.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,59 @@
+name: 🐞 Bug Report
+description: Report a bug
+labels: 'type: bug'
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ## First of all
+        1. Please search for [open issues](https://www.github.com/tauri-apps/tauri/issues) about this problem first.
```

Comment:
can we also add a note about discord here? and github discussions as well and also search closed issues is sometimes useful.

Reply:
Would love to keep it as short as possible. I'll change `open issues` to `existing issues`. The Discord link can hopefully be in `config.yml` like you suggested.

Reply:
cool